x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ClientHelper.java

```diff
@@ -190,6 +190,8 @@ private static String maybeRewriteSingleAuthenticationHeaderForVersion(
     public static final String LOGSTASH_MANAGEMENT_ORIGIN = "logstash_management";
     public static final String FLEET_ORIGIN = "fleet";
 
+    public static final String BEHAVIORAL_ANALYTICS_ORIGIN = "behavioral_analytics";
```

Comment:
❓ Do we want an origin for each big feature (analytics, engine, ...) or a single one for the module.

My assumption is that we want one for each big feature so we can break the module in several more easily if it become too big or we want the functional boundaries to be more clear.